configure.ac

```diff
@@ -639,6 +664,14 @@ if test x$use_pkgconfig = xyes; then
   ])
 fi
 
+dnl Find the additional lib (if any) responsible for handling
```

Comment:
Do we need this for anything other than Rust?

Reply:
No, and I'm not actually sure why we need it for rust. It would be nice if we could disable the functionality. Again, maybe this isn't necessary now that we've slimmed down and aren't using cargo.

Reply:
This also seems to be unnecessary now, at least for linux/win/osx. After removing it (as well as manually removing -ldl from the link line, which some library has rudely added), builds still succeed.

Reply:
Oops, looks like we do - I unset this then got a build failure on Debian Stable. I did move the block into the if use_rust block.